crates/codegen/runtime/cargo/wasm/src/runtime/interface/cst.wit.jinja2

```diff
@@ -11,7 +11,14 @@ interface cst {
             stub3,
         {%- else %}
             {%- for variant in model.kinds.nonterminal_kinds %}
-                {{ variant | wit_case }},
+                /// This kind represents a `{{ variant.id }}` node, with the following structure: 
+                ///
+                /// ```ebnf
+                {%- for line in variant.documentation | split(pat="\n") %}
+                /// {{ line }}
+                {%- endfor %}
```

Comment:
I wonder if this has an advantage over the `indent()` syntax that was already used?

```
{{ variant.documentation | indent(prefix = "/// ", first = true, blank = true) }}
```

Reply:
`indent()` put any documentation lines after the first at the beginning of the line, with no leading whitespace before `///`. This was the only way to output the doc comments inline with each other.

Reply:
Thanks for explaining! maybe we should fix it in the other site as well:

- `cst/terminal_kind.rs.jinja2`
- `cst/nonterminal_kind.rs.jinja2`

Reply:
It doesn't need to be fixed there because `rustfmt` fixes it after it's generated.